test/adapter_handle_test.c

```diff
@@ -0,0 +1,41 @@
+/* This program is released under the Common Public License V1.0
+ *
+ * You should have received a copy of Common Public License V1.0 along with
+ * with this program.
+ */
+
+/* Copyright IBM Corp. 2022 */
+
+#include <stdio.h>
+#include <sys/errno.h>
```

Comment:
Why not simply `#include <errno.h>`?

Reply:
hm, good question. Seems like a copy&paste leftover. I'll fix it.